locales/fr/translation.json

```
@@ -303,7 +303,7 @@
   "html.basics-no-uploaded-data": "<0>La vue de base affiche un résumé des récentes activités de votre système, mais il semblerait que votre système n'a pas encore remonté de données. </0><1>Pour visualiser la vue de base, <1>Assurez-vous que votre système est actif</1>. </1><2>Si vous venez de l'activer, essayer de <1>rafraîchir la page</1>.</2>",
   "html.bg-log-no-uploaded-data": "La vue hedbomadaire montre l'historique de vos glycémies capillaires, mais il semblerait que votre système n'ait pas encore remonté de telles données.</0><1><1>Vérifiez</1> que votre système est actif.</1><2>Si vous venez de le faire, essayez de <1>rafraîchir la page</1>.</2>",
   "html.browser-warning-text": "<0>Copier - coller </0><1></1><2>dans chrome.</2>",
-  "html.cookie-content": "Merci de consulter notre <1>politique de confidentialité</1> et nos <3>conditions d'utilisations</3>",
+  "html.cookie-content": "Please consult our <1>Privacy policy</1> and our <3>Terms of Use</3>",
```

on a perdu la bonne langue ?
```suggestion
  "html.cookie-content": "Merci de consulter notre <1>Politique de Confidentialité</1> et nos <3>Conditions d'Utilisations</3>",
```